app/models/feed.rb

```diff
@@ -41,6 +41,38 @@ def set_defaults
     self.enclosure_template ||= Feed.enclosure_template_default
   end
 
+  def represented_tokens
+    feed_tokens.map do |t|
+      {
+        label: t.label,
+        token: t.token,
+        expires: t.expires_at
+      }.delete_if { |k, v| v.nil? }
+    end
+  end
+
+  def represented_tokens=(updates)
```

Comment:
As I discovered in Augury - using a nested-collection setter like this doesn't play well with Representable/Roar/HalApi stuff.

So the easiest path is to bypass that entirely, and just pretend this list-of-hashes is a simple property on the Feed.

Reply:
I think the issue I had in Augury was related to our HalApiRails overrides.  I had some [notes here](https://github.com/PRX/augury.prx.org/blob/master/app/representers/api/flight_representer.rb#L22) ... and figured I'd just avoid the issue entirely here.

Yeah, if we do figure that out, at least the tests will stay the same!